src/lib/shared/utils.ts

```diff
@@ -98,17 +98,14 @@ export function base64StringToTransaction<T extends Transaction>(transactionByte
  * @param transaction - a base64 encoded string of proto.TransactionBody.encode().finish()
  * @returns `string`
  * */
-export function transactionToTransactionBody<T extends Transaction>(
-  transaction: T,
-  nodeAccountId: AccountId,
-) {
+export function transactionToTransactionBody<T extends Transaction>(transaction: T) {
```

Comment:
I'm uncomfortable changing the signature of a public method, this could result in a breaking change to existing implementations.

Reply:
Maybe this implementation should remain and `Signer.ts` changed from 

```async signTransaction<T extends Transaction>(transaction: T): Promise<T> {
    const transactionBody: proto.TransactionBody = transactionToTransactionBody(
      transaction,
      this._getRandomNodes(1)[0],
    )
```

to re-using the `nodeAccountId` from `transaction` (if that's possible) ?

This would no longer be a breaking change.

Reply:
Thanks for the feedback! 🙂

From our testing and research, the method used in this PR is the only way to get the signed transaction validated by the Hedera network. The previous approach signed the transaction in a way that the Hedera Network couldn't verify.

That said, we can leave the nodeAccountId parameter for compatibility, although it's currently unused and likely not functional in its previous form. @gregscullard, should I include the nodeAccountId as you suggested?

If there is a better method to obtain the signed transaction bytes than the one used in this PR, please let me know. Based on our testing a few weeks ago, this was the only approach that worked.

Reply:
I'll leave it to the community of builders/users to determine whether a breaking change in the signature of the transaction is acceptable or not. The change is minor and would likely be implemented as a result of changes to the application, so tested and working before deployed. Breaking but not critically so I guess.